src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1897,64 +1956,79 @@ private long EnforceConfigurationFor(DocumentsOperationContext context, string i
             }
         }
 
-        private class EnforceRevisionConfigurationCommand : TransactionOperationsMerger.MergedTransactionCommand
+        internal long AdoptOrphanedFor(DocumentsOperationContext context, string id)
         {
-            private readonly RevisionsStorage _revisionsStorage;
-            private readonly List<string> _ids;
-            private readonly bool _includeForceCreatedRevisionsOnDeleteInCaseOfNoConfiguration;
-            private readonly EnforceConfigurationResult _result;
-            private readonly OperationCancelToken _token;
+            using (DocumentIdWorker.GetSliceFromId(context, id, out var lowerId))
+            using (GetKeyPrefix(context, lowerId, out var lowerIdPrefix))
+            {
+                var collectionName = GetCollectionFor(context, lowerIdPrefix);
+                if (collectionName == null)
+                {
+                    if (_logger.IsInfoEnabled)
+                        _logger.Info($"Tried to delete revisions for '{id}' but no revisions found.");
```

Comment:
to adopt?
and "no collection was found"?

Reply:
Done